README.md

```diff
@@ -1,61 +1,75 @@
-long tasks can help keep an app responisive when also *yielding to the browser's
-event loop*.
+Dealing with contention is largely a scheduling problem: to the degree that
+work can be reordered in an more optimal way, scheduling can have a positive
+impact. What makes this problem more pronounced on the web is that tasks run to
+completion, meaning the UA cannot interrupt a task to run high priority work
+like processing user input. This problem is generally tackled in userspace by
+systematically chunking and scheduling main-thread work. Since long tasks and
+responsiveness are at odds, breaking up long tasks can help keep an app
+responsive when also *yielding to the browser's event loop*.
 
-[Userspace schedulers](./misc/userspace-schedulers.md) have evolved to manage these
-chunks of work&mdash;prioritizing and executing work async at an appropriate
-time relative to current situation of user and browser.
-
-While userspace schedulers have been effective in improving responsiveness,
-there are several problems they still face:
+[Userspace schedulers](./misc/userspace-schedulers.md) have evolved to manage
+these chunks of work&mdash;prioritizing and executing work async at an
+appropriate time relative to current situation of user and browser. And while
+userspace schedulers have been effective in improving responsiveness, there are
+several problems they still face:
 
  1. **Determining when to yield to the browser**: yielding has overhead&mdash;the
     overhead of posting a task and context switching, the cost of regaining
-    control, etc. This leads to increased task latency (the task at hand takes
-    longer).
+    control, etc. This can lead to increased task latency.
 
-    Making intelligent decisions of when to yield is difficult with limited
+    Making intelligent decisions about when to yield is difficult with limited
     knowledge. Scheduling primitives can help userspace schedulers make better
     decisions, e.g. [isInputPending()](https://github.com/WICG/is-input-pending)
-    and isFramePending().
+    and [isFramePending()](https://github.com/szager-chromium/isFramePending/blob/master/explainer.md).
 
  2. **Regaining control after yielding**: chunking work and yielding is
     necessary for improving responsiveness, but it comes at a cost: when
     yielding to the event loop, a task that yields has no way to *continue*
     without arbitrary work of the *same priority* running first, e.g. other
     script. This disincentivizes yielding for script that requires low task
-    latency.
-
-    We propose adding [scheduler.yield()](./explainers/yield-and-continuation.md) as a
-    solution.
+    latency. Providing a primitive like [scheduler.yield()](./explainers/yield-and-continuation.md)
+    that is designed to take into account this async userspace task model can
+    help, as the scheduler can prioritize these continuations more fairly.
 
  3. **Coordination between (cooperating) actors**: Most userspace schedulers
```

Comment:
Consider making this the first item so people don't ask why we're doing the third-most-important thing first.